bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/tracer/TraceConstants.java

```diff
@@ -35,4 +35,7 @@ private TraceConstants() {
 
     public static final String TAG_SPAN_KIND_SERVER = "server";
     public static final String TAG_SPAN_KIND_CLIENT = "client";
+
+    public static final String TRACE_ID = "traceId";
+    public static final String SPAN_ID = "spanId";
```

Comment:
Shall we rename these to indicate these are for span context keys ? We can also make these BStrings directly.
```suggestion
    public static final BString SPAN_CONTEXT_MAP_KEY_TRACE_ID = StringUtils.fromString("traceId");
    public static final BString SPAN_CONTEXT_MAP_KEY_SPAN_ID = StringUtils.fromString("spanId");
```

Reply:
changed